relay-profiling/Cargo.toml

```diff
@@ -14,9 +14,11 @@ android_trace_log = { version = "0.3.0", features = ["serde"] }
 chrono = { workspace = true }
 data-encoding = "2.3.3"
 relay-event-schema = { path = "../relay-event-schema" }
+relay-log = { path = "../relay-log" }
 relay-protocol = { path = "../relay-protocol" }
 serde = { workspace = true }
 serde_json = { workspace = true }
+serde_path_to_error = "0.1.14"
```

Comment:
Code of this library looks OK to me. Given it has the same author & license as serde, I think we can include it.